plugins/nodes/src/js/pages/nodes/nodes-table/NodesTableContainer.js

```diff
@@ -27,6 +27,15 @@ class NodesTableContainer extends mixin(StoreMixin, QueryParamsMixin) {
       },
       { name: "state", events: ["success"], suppressUpdate: false }
     ];
+
+    setInterval(this.messIt.bind(this), 1000);
+  }
+
+  messIt() {
+    const items = this.state.filteredNodes.list;
```

Comment:
Instead `this.state.filteredNodes.list` I would suggest `this.state.filteredNodes.list.slice(0)` that way avoiding shallow copy.

Reply:
I think this is intended, or at least it helps me to cause the bug I want to make happen. I will try to remove and see if it causes the bug.